web/components/TeamMember.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import Flex from './Flex';
+import Text from './Text';
+import Card from './Card';
+import Image from './Image';
+
+
+export default class TeamMember extends React.Component {
+    render() {
+        const { goToPage, name, color } = this.props;
+        return (
+            <>
+                <Card
+                    bg={color}
+                    borderRadius={5}
+                    width='70%'
+                    minWidth='20%'
+                    maxWidth='530px'
+                    height='30px'
+                    mx='auto'
+                    my='1rem'
+                    depth='-1px 0px 15px rgba(0,0,0,0.65)'
```

Comment:
Use `shadow` property (https://github.com/setlife-network/setblocks/blob/master/web/styles/theme.js#L13)

The `depth` property should be 1-5 so that it uses the standardized box shadow styles above (https://github.com/setlife-network/setblocks/blob/master/web/components/Card.js#L40)